src/services/OutbreakService/OutbreakService.spec.ts

```diff
@@ -330,4 +330,21 @@ describe('OutbreakService', () => {
     const outbreakHistory = service.outbreakHistory.get();
     expect(outbreakHistory).toHaveLength(1);
   });
+
+  it('auto deletes checkin after CHECKIN_NOTIFICATION_CYCLE', async () => {
+    await service.addCheckIn(checkIns[4]);
+    await service.addCheckIn(checkIns[5]);
+
+    // day 1
+    MockDate.set('2021-01-01T10:00Z');
```

Comment:
I changed the date on this to 2021-02-08, and it is still passing. The time stamps on the ID is 2021-01-01, shouldn't it fail?